src/reporters/markdownReporter.ts

```diff
@@ -4,7 +4,7 @@ import { groupBy } from "../action/groupBy.js";
 import { formatReportAsMarkdown } from "./formatReportAsMarkdown.js";
 
 export const markdownReportPassMessage =
-	"All reports are resolved now. Thanks! ✅";
+	"All reports are resolved now. Thanks! ✅\n\n<!-- resolved-by: OctoGuide -->";
```

Comment:
[Refactor] This `resolved-by: OctoGuide` text is pretty important. It's also hardcoded twice. What do you think about extracting it to some shared variable?